src/Admin/AbstractAdmin.php

```diff
@@ -2692,12 +2692,7 @@ public function getPermissionsShow($context)
 
     public function showIn($context)
     {
-        switch ($context) {
-            case self::CONTEXT_DASHBOARD:
-            case self::CONTEXT_MENU:
-            default:
-                return $this->isGranted($this->getPermissionsShow($context));
-        }
+        return $this->isGranted($this->getPermissionsShow($context));
```

Comment:
Can you please explain this?

Reply:
I may be wrong but if you look at the code, the switch is useless.

Reply:
Ok, I trust you.